Sources/iosDesignTokens/todaytix/GradientTokens.swift

```diff
@@ -0,0 +1,149 @@
+
+// Do not edit directly
+// Generated on Thu, 10 Nov 2022 19:19:40 GMT
+
+
+import UIKit
+
+public class GradientTokens {
+	public var lightInteractiveInteractive01: CAGradientLayer {
```

Comment:
I am not sure what this is based on the name

Reply:
it's based on the token path. When we generate the token files with amazon style dictionary, this is the name it gives us. Sean is working on naming so this isn't necessarily final or anything - I just needed something to test with.